### PR TITLE
use correct optimizer object

### PR DESCRIPTION
**Pull request by jbinas**
The sgd object created was not passed to the model, so the actual sgd parameters used were the default ones (lr=0.01, momentum=0.0, decay=0.0, nesterov=False).

### CODE REVIEW

**Comment by iabhi7** [comment]
Hi @jbinas really appreciate your PR
However, I would suggest you to change the LR to 0.1 because for me it took 350 epoch and almost a day to converge (that to in a g2.8xlarge AWS instance). For normal users, it would be encouraging if they see good results just after few epochs and then experiment their way out.

**Comment by jbinas** [comment]
I believe that you might have been using a learning rate of 0.01 rather than 0.1, too (at least this is the case for the code you provide, since lr=0.01 is the default in Keras and lr=0.1 is never set in your code).
In fact, lr=0.01 gives me roughly the results you report, whereas lr=0.1 does not seem to work at all.
